index.html

```diff
@@ -15,64 +15,159 @@
 
       <!-- Navigation bar section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Intro section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- What is section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Atomist enables, cards section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Powered by atomist section -->
       <div class="row">
-        <div class="column">
+        <div class="col">
 
         </div>
       </div>
 
 
       <!-- Latest from atomist section -->
       <div class="row">
-        <div class="column">
-
+        <div class="col">
+          <h3>The Latest from Atomist</h3>
+          <div class="card">
+            <img src="wordcloud.jpg" alt="">
```

Comment:
If these images are local, we won't be able to see them if we merge to dev.  Try using a placeholder image like https://placekitten.com/ or add them to the `src/assets` folder and then change the source in the `img` tag.

Reply:
This is the only change I would like to see before merging, the other ones can be in the other sprint. It applies to all the added images.

Reply:
I'll add them to the src/assets folder,  I wasn't careful enough before, thanks for the feedback!